io/net/netperf_test.py

```diff
@@ -70,7 +70,7 @@ def setUp(self):
             self.cancel("failed connecting to peer")
         smm = SoftwareManager()
         detected_distro = distro.detect()
-        pkgs = ['gcc']
+        pkgs = ['gcc', 'unzip']
```

Comment:
@nick-child-ibm is this install required for host or peer?
if it is required to be installed on peer, then are we handling this in the script?
please add the the logs after the fix is happening.

Reply:
> Is this install required for host or peer or both?
Install appears to be done on both host and peer. But the host uses the `archive` avocado.utils module instead of the `unzip` command. So only peer depends on `unzip`.

>  if it is required to be installed on peer or on both, then are we handling this in the script?
Yes, the following code already checks for the dependency and installs if necessary:
```
        for pkg in pkgs:
            if not smm.check_installed(pkg) and not smm.install(pkg):
                self.cancel("%s package is need to test" % pkg)
            cmd = "%s install %s" % (smm.backend.base_command, pkg)
            output = self.session.cmd(cmd)
            if not output.exit_status == 0:
                self.cancel("unable to install the package %s on peer machine "
                            % pkg)
  ```
 >  is this install part still manual on peer?
 Not sure what you mean by manual. `SoftwareManager` functions are used for installing.
 
 I will add logs shortly.